etl/steps/data/garden/demography/2022-12-08/population/meta.yml

```diff
@@ -59,13 +63,18 @@ tables:
         description: |
           Share of the world's population by country, available from 10,000 BCE to 2100 based on Gapminder data, HYDE, and UN Population Division (2022) estimates.
 
-          10,000 BCE - 1799: Historical estimates by HYDE (v3.2).
-          1800-1949: Historical estimates by Gapminder.
-          1950-2021: Population records by the United Nations - Population Division (2022).
-          2022-2100: Projections based on Medium variant by the United Nations - Population Division (2022).
+          - 10,000 BCE - 1799: Historical estimates by HYDE (v3.2). Includes some datapoints Gapminder (Systema Globalis).
+          - 1800-1949: Historical estimates by Gapminder. Includes some datapoints from HYDE (v3.2) and Gapminder (Systema Globalis).
+          - 1950-2021: Population records by the United Nations - Population Division (2022). Includes some datapoints from HYDE (v3.2), Gapminder (Systema Globalis) and Gapminder (v6).
+          - 2022-2100: Projections based on Medium variant by the United Nations - Population Division (2022).
         unit: "%"
         short_unit: "%"
         display:
           name: Share of world population
           includeInTable: true
         sources: *population-sources
+      source:
```

Comment:
Isn't this column dropped? (In line 69 of the step)

Reply:
This column is dropped in the Grapher step, but it is still available in Garden.